ansible/group_vars/all.yml

```diff
@@ -36,7 +36,7 @@ virtual_machines:
   rhel-s390x:
     project: rhel-s390x-cloud
     arch: s390x
-    ssh_key_file: ibmcloud_ssh_key_file
+    ssh_key_file: "{{ ibmcloud_ssh_key_file }}"
```

Comment:
Dang! My bad

Reply:
and my bad for missing it in the review 😄